.circleci/images/70-build-collector-slim-and-base-images.sh

```diff
@@ -23,11 +23,13 @@ fi
     "-" \
     "$SOURCE_ROOT/${COLLECTOR_BUILD_CONTEXT}"
 
+make -C "$SOURCE_ROOT" container-dockerfile
```

Comment:
Looks like your PR is pointing to an old version of master.  Please run `smart-rebase` on your branch and force push the changes.

Reply:
I thought I had rebased. It is rebased again.